src/pages/FavoritePage.js

```diff
@@ -1,125 +1,35 @@
 import React from 'react'
 import Header from '../components/Header'
-import Bookmark from '../components/Bookmark'
-import CardBody from '../components/CardBody'
+import CardList from '../components/CardList'
+import { useLocation } from 'react-router-dom'
 import styled from 'styled-components/macro'
-import { useToggle } from 'react-hooks-lib'
-import defaultImageSrc from '../imgsrc/unknownProfileImg.png'
 
-export default function FavoritePage({ profiles, onBookmarkClick }) {
+export default function FavoritePage({ profiles, onBookmarkClick, onDelete }) {
   const filterProfiles = profiles.filter(
```

Comment:
Maybe rename to `filteredProfiles`? `filterProfiles` sounds like a function name, I think.